src/corppa/utils/filter.py

```diff
@@ -40,17 +40,27 @@ def filter_pages(
     exclude_filter: dict | None = None,
 ) -> Iterator[dict]:
     """Takes a filename for a PPA full-text corpus in a format orjsonl supports
-    and a list of source ids. Returns a generator of filtered pages from the
-    full corpus corresponding to the list of ids.  Displays progress
-    with :mod:`tqdm` progress bar unless disabled.
+    and one or more options for filtering that corpus. Returns a generator of
+    filtered pages from the full corpus corresponding to the list of ids.
+    At least one filtering option must be specified.
+    Displays progress with :mod:`tqdm` progress bar unless disabled.
 
     :param input_filename: str, filename for corpus input
     :param source_ids: list of str, source ids to include in filtered pages
+    :param include_filter: dict of key-value pairs for pages to include in
+        the filtered page set; equality check against page data attributes (optional)
+    :param exclude_filter: dict of key-value pairs for pages to exclude from
+        the filtered page set; equality check against page data attributes (optional)
     :param disable_progress: boolean, disable progress bar (optional, default: False)
     :returns: generator of dict with page data
     :raises: FileNotFoundError, orjson.JSONDecodeError
     """
     # convert list of source ids to set for fast hashmap lookup
```

Comment:
Relocate this comment to associated code (above L64). Consider adding the following comment for the input parameter check, since it's not documented in the method-level comment (taken from `save_filtered_corpus`)

Reply:
```suggestion
    # at least one filter is required
```